app/src/main/java/com/example/nostack/services/NavbarConfig.java

```diff
@@ -148,4 +148,14 @@ public void setVisible(){
     public void setInvisible(){
         navbar.setVisibility(navbar.INVISIBLE);
     }
+
+    /**
+     * Set the icon of the navbar Floating Action Button to Camera
+     */
+    public void setIconCamera() {scanQRButton.setImageResource(R.drawable.navbarcamera);}
```

Comment:
this can go into the individual functions (Attendee, Orgnizer, Admin)

Reply:
![image](https://github.com/CMPUT301W24T27/NoStack/assets/36091727/fae8d32c-ac6d-4e67-902c-e730380b9ee9)